bundles/org.openhab.binding.speedtest/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,158 @@
+
+			<channel id="downloadBandwidth" typeId="downloadBandwidth"/>
+			<channel id="downloadBytes" typeId="downloadBytes"/>
+			<channel id="downloadElapsed" typeId="downloadElapsed"/>
+
+			<channel id="uploadBandwidth" typeId="uploadBandwidth"/>
+			<channel id="uploadBytes" typeId="uploadBytes"/>
+			<channel id="uploadElapsed" typeId="uploadElapsed"/>
+
+			<channel id="isp" typeId="isp"/>
+
+			<channel id="interfaceInternalIp" typeId="interfaceInternalIp"/>
+			<channel id="interfaceExternalIp" typeId="interfaceExternalIp"/>
+
+			<channel id="resultUrl" typeId="resultUrl"/>
+
+			<channel id="triggerTest" typeId="triggerTest"/>
+		</channels>
+		<properties>
+			<property name="Server List 1"></property>
+			<property name="Server List 2"></property>
+			<property name="Server List 3"></property>
+			<property name="Server List 4"></property>
+			<property name="Server List 5"></property>
+			<property name="Server List 6"></property>
+			<property name="Server List 7"></property>
+			<property name="Server List 8"></property>
+			<property name="Server List 9"></property>
+			<property name="Server List 10"></property>
+		</properties>
+		<config-description>
+			<parameter-group name="config-info">
+				<label>Speedtest Configuration</label>
+			</parameter-group>
+			<parameter name="refreshInterval" type="integer" required="true" groupName="config-info">
+				<label>Refresh Rate</label>
+				<description>How often to test network speed</description>
+				<options>
+					<option value="15">15 minutes</option>
+					<option value="30">30 minutes</option>
+					<option value="60">Every Hour</option>
+					<option value="360">Every 6 Hours</option>
+					<option value="720">Every 12 Hours</option>
+					<option value="1440">Once a Day</option>
+					<option value="0">Don't test automatically</option>
+				</options>
+				<default>60</default>
+			</parameter>
+			<parameter name="execPath" type="text" required="false" groupName="config-info">
+				<label>Speedtest Path</label>
+				<description>The path of the Ookla Speedtest executable. Linux machines may leave this blank and it defaults to
+					`/usr/bin/speedtest`.</description>
+			</parameter>
+			<parameter name="serverID" type="text" required="false" groupName="config-info">
+				<label>Server ID</label>
+				<description>Optional: A specific server that shall be used for testing. You can pick the server ID from the "Thing
+					Properties". If this is left blank the best option will be selected by Ookla.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-type id="server">
+		<item-type>String</item-type>
+		<label>Server</label>
+		<description>The remote server that the Speedtest was run against</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="pingJitter" advanced="true">
+		<item-type>Number:Time</item-type>
+		<label>Ping Jitter</label>
+		<description>Ping Jitter</description>
```

Comment:
Can you give a little more info so its not the same as the name and title? 1 way, round trip, perhaps mention its in milliseconds etc.

Reply:
done - please have a look